### PR TITLE
Update readme for running manager locally

### PR DESCRIPTION
**Pull request by ityuhui**
This PR adds environment variables about Razee to the readme.

### CODE REVIEW

File: manager/README.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ VAULT_MODULES_ROLE="module"
 ENABLE_WEBHOOKS="false"
 ```
 
+If the manager runs in the Razee environment, you also need to add the following environment variables:
```
[the remark this answers is not in the document]
Comment (ityuhui) [reply manager/README.md:58]: When a Razee environment is setup:

1. There are "razeedash" and "razeedash-api" pods running at fybrik control plane:
`kubectl get po -A | grep razeedash`

2. Can get response from NodePort service of "razeedash-api-np"
`curl http://localhost:3333/graphql`

File: manager/README.md [hunk 2]
```diff
@@ -55,6 +55,14 @@ VAULT_MODULES_ROLE="module"
...
+
+```bash
+RAZEE_USER="razee-dev@example.com"
+RAZEE_PASSWORD="password123"
+RAZEE_URL="http://localhost:3333/graphql"
```
Comment (shlomitk1) [inline manager/README.md:63]: Where is the port-forward from razee pod to localhost:3333?

Comment (ityuhui) [reply manager/README.md:63]: Don't need a "port-forward".
There is a NodePort service "razeedash-api-np" created by https://github.com/fybrik/fybrik/blob/ee85999dd766b97feb0ab2f7e095247deccb580f/third_party/razee/razee-nodeports.yaml#L5

Comment (shlomitk1) [reply manager/README.md:63]: OK, thanks!

File: manager/README.md [hunk 3]
```diff
@@ -55,6 +55,22 @@ VAULT_MODULES_ROLE="module"
 ENABLE_WEBHOOKS="false"
 ```
 
+If the manager runs in a Razee environment, you also need to add the following environment variables:
```
Comment (roytman) [inline manager/README.md:58]: Hi thanks for the PR, 
but it is not clear what do you mean by "a Razee environment", I assume that you mean Razee and not Satellite deployment. However, the Raze can be collocated on the same cluster as the manager, or can be a remote service. Manager can connect to Razee by using userName/password  as you explain here or by using oauth API key, as shown [here](https://fybrik.io/v0.6/tasks/multicluster/#coordinator-cluster).

When a user (developer)  runs  manager manually, he probably knows knows which Razee he plans to use. 
I would suggest the following changes:
If the manager will works with a Razee service,  you also need to add the following environment variables:
```bash
RAZEE_URL=<Razee access point> # e.g. "http://localhost:3333/graphql"
RAZEE_USER=<Razee user>
# you should define either RAZEE_PASSWORD or API_KEY, if both variables are defined RAZEE_PASSWORD will be used
RAZEE_PASSWORD=<Razee password> 
API_KEY=<Razee api key>
```
And the paragraph `You can know that the manager runs in a Razee environment by` (Lines 66- 73) can be removed.

Comment (ityuhui) [reply manager/README.md:58]: Thanks! I will update it with your suggestions.

**Review by roytman — APPROVED** [review]

**Pull request merged**